### ISSUE TITLE
udp support is part of the roadmap?

### ISSUE THREAD

**Issue by hiqsociety** [monoio-rs/monoio#64]
udp support is part of the roadmap?
trying to use s2n-quic with it

**Comment by ihciah** [comment]
Yes. UDP will be supported soon(maybe also with AF_XDP support).

**Comment by Kiddinglife** [comment]
udp +1

**Comment by hiqsociety** [comment]
@ihciah when will udp (maybe with af_xdp) support happen?
looking at interfacing with http3 mostly.

**Comment by joeky888** [comment]
A KCP server/client example would be nice once udpsocket is done. :)
